src/nunit-gui/Presenters/TreeViewPresenter.cs

```diff
@@ -97,8 +97,13 @@ private void WireUpEvents()
             };
 
             // View context commands
-            _view.Tree.ContextMenu.Popup += () =>
-                _view.RunCheckedCommand.Visible = _view.Tree.CheckBoxes && _view.Tree.CheckedNodes.Count > 0;
+            _view.Tree.ContextMenu.Popup += delegate
```

Comment:
Here you use the delegate keyword and below you use a statement labmda. Since this appears to be the only one that uses delegate I suggest changing it to a statement lambda.